kse/src/org/kse/crypto/privatekey/PrivateUtils.java

```diff
@@ -0,0 +1,222 @@
+package org.kse.crypto.privatekey;
```

Comment:
There currently is no existing package where this class would fit in. It contains UI code, so it has to be under `org.kse.gui.crypto`, so just create a new package `privatekey` there.